diff --git a/pico-sdk/tools/elf2uf2/main.cpp b/tools/elf2uf2/main.cpp
index 07cd107..fd4fb9d 100644
--- a/pico-sdk/tools/elf2uf2/main.cpp
+++ b/pico-sdk/tools/elf2uf2/main.cpp
@@ -268,7 +268,7 @@ int elf2uf2(FILE *in, FILE *out) {
         if (eh.entry == expected_ep_xip_sram) {
             return fail(ERROR_INCOMPATIBLE, "B0/B1 Boot ROM does not support direct entry into XIP_SRAM\n");
         } else if (eh.entry != expected_ep) {
-            return fail(ERROR_INCOMPATIBLE, "A RAM binary should have an entry point at the beginning: %08x (not %08x)\n", expected_ep, eh.entry);
+            //return fail(ERROR_INCOMPATIBLE, "A RAM binary should have an entry point at the beginning: %08x (not %08x)\n", expected_ep, eh.entry);
         }
         static_assert(0 == (MAIN_RAM_START & (PAGE_SIZE - 1)), "");
         // currently don't require this as entry point is now at the start, we don't know where reset vector is
